crates/subspace-farmer/src/single_plot_farm.rs

```diff
@@ -225,18 +238,19 @@ impl SinglePlotFarm {
         })
     }
 
-    pub(crate) fn dsn_sync(
+    pub fn dsn_sync(
```

Comment:
Why does it need to be public? Looks like implementation detail to me that doesn't need to be exposed.

Reply:
Hmm, we would either need to put `bench` stuff and this new test in the library itself or make public some functions and structures.

I think it would be better to move benchmarking utils under the farmer library.

Reply:
Moved benchmarking utils under the farmer library